src/electron/main.ts

```diff
@@ -10,7 +10,7 @@ import LocalFileSystem from "./providers/storage/localFileSystem";
 let mainWindow: BrowserWindow;
 let ipcMainProxy: IpcMainProxy;
 
-function createWindow() {
+async function createWindow() {
```

Comment:
Why making `createWindow` async? I don't see any calls to async methods.

Reply:
Forgot to restore this after some testing with Electron.Remote